odk1-src/central-install.rst

```diff
@@ -0,0 +1,34 @@
+.. _central-install:
+
+Installing ODK Central
+======================
+
+Unlike ODK Aggregate, Central does not run on Google App Engine. Instead, it is built on a new technology called `Docker <https://en.wikipedia.org/wiki/Docker_(software)>`_, which is a new kind of virtual machine (VM) system that allows smaller image files, better performance, and better ways to define how different pieces should fit together to create a server like Central. Don't worry if you don't know what that means! We have put together step-by-step instructions for our recommended solutions below:
+
+.. _central-install-sandbox:
+
+Using the Sandbox
+-----------------
+
+If you only want to try ODK Central to see if it's suitable for your project, consider skipping installation altogether and using the `Sandbox installation <https://sandbox.central.opendatakit.org/>`_ we've provided for exactly this reason. Do note that since there is only one Sandbox, all Sandbox users will be able to see each others' email addresses, form definitions, and submission data, so please be careful if you have sensitive information you wish to keep secret.
+
+Otherwise, private message `Yaw Anokwa on the Open Data Kit Forum <https://forum.opendatakit.org/u/yanokwa>`_ to gain access to the Sandbox.
```

Comment:
The otherwise is a little wierd here. I'd drop it.

It's weird to see my name here.  Maybe `private message @yanokwa on the ODK Forum`.